python/pyraphtory/pyraphtory/context.py

```diff
@@ -23,6 +27,7 @@
 from pyraphtory import interop
 import sys
 
+VERSION = '0.2.0a0'
```

Comment:
versioning should be external? like from an env variable or from a file?

Reply:
can you recommend anything? I have a version file in the base package for raphtory, but that is inaccessible in python. I can do the same, where we have a version file in the python root dir?

Reply:
okay now the python script gets the version from the toml file. note that toml is static and therefore it cannot load its own version .